src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -292,8 +295,6 @@ public void TrackScreenView(string viewName)
 
         public void TrackException(Exception ex, bool isFatal)
         {
-            if (!ReportingAnalytics) return;
-
```

Comment:
@QilongTang do you know if this is intentional ?
Exception tracking seems to have been enabled regardless of Analytics Opt-in status.

Reply:
It is even covered by a test...

Reply:
@pinzart Wait doesn't think mean if not reporting then simply return here?

Reply:
Yes. (the code that I added, as part of ADP work).
The code before my changes did not have that condition...so it reported exceptions even if the user was opted out

Reply:
I want to know if that was a feature request ... or if I can leave my changes and update the existing test..